wandb/sdk/launch/_project_spec.py

```diff
@@ -87,11 +87,7 @@ def __init__(
         self._entry_points: Dict[
             str, EntryPoint
         ] = {}  # todo: keep multiple entrypoint support?
-        if (
-            "entry_point" in overrides
-            and overrides["entry_point"] is not None
-            and overrides["entry_point"] != ""
-        ):
+        if "entry_point" in overrides and overrides["entry_point"]:
```

Comment:
I think `is not None` is more robust. Kinda sucks that it is extra words, but it prevents the `overrides: { entrypoint: True}` edge case

Reply:
Hmm, yeah sounds good.